src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double MAX_SIZE = 11.0;

```

Comment:
The constant MAX_SIZE is better to be named indicating its purpose, like MAX_FIGURE_SIZE, for better readability and understanding of the code.